_component_design/buttons.md

```diff
@@ -14,203 +14,193 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-Buttons provide the mechanism for users to perform actions within a
-Helix-themed UI. Each button has a weight (Primary, Secondary, and Tertiary),
-a size (Regular, Large, Compact), as well as a type (Simple, Dropdown).
+## Introduction
+{: .hxSectionTitle}
 
-{% figure [caption:"The three types of buttons defined in the Helix design language"] [class:"image bg-light border col-md-12"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-types.svg)
-{% endfigure %}
-
-# Primary Buttons
+Buttons provide the mechanism for users to perform actions within a Helix UI. Each button has a weight (Primary, Secondary, and Tertiary), a size (Regular, Large, Compact), as well as a type (Simple, Dropdown).
```

Comment:
`dropdown` should be `drop-down` per style.

Reply:
7 instances updated across doc.